### PR TITLE
fixed referral id on get

### PR DESCRIPTION
**Pull request by severinbeauvais**
I also deleted an obsolete referral model variable.
Tested locally!

### CODE REVIEW

File: embc-app/ClientApp/src/app/volunteer/pages/referral-view/referral-view.component.ts [hunk 1]
```diff
@@ -58,7 +58,9 @@ export class ReferralViewComponent implements OnInit, OnDestroy {
           console.log('ERROR - invalid referral object = ', x);
           this.goHome();
         } else {
-          this.referral = x.referral;
+          // HACK for BE returning id instead of referralId
```
Comment (ytqsl) [inline embc-app/ClientApp/src/app/volunteer/pages/referral-view/referral-view.component.ts]: This was changed on the FE request, do you prefer the BE will be changed back to referralId?

Comment (severinbeauvais) [reply embc-app/ClientApp/src/app/volunteer/pages/referral-view/referral-view.component.ts]: That's a valid option too. Just LMK what you do so I can keep up :)

Comment (ytqsl) [reply embc-app/ClientApp/src/app/volunteer/pages/referral-view/referral-view.component.ts]: your call (BFF)

Comment (LayCraft) [reply embc-app/ClientApp/src/app/volunteer/pages/referral-view/referral-view.component.ts]: At the time of the request it was to work around a bug in a component built on the original data that I didn't have time to fix. I think that bug is fixed? If we pick a consistent way then I will be fine with whatever.
I'll wait for Severin to weight in.

Comment (severinbeauvais) [reply embc-app/ClientApp/src/app/volunteer/pages/referral-view/referral-view.component.ts]: Let's not make further changes now. We can change this later.

Comment (severinbeauvais) [reply embc-app/ClientApp/src/app/volunteer/pages/referral-view/referral-view.component.ts]: > your call (BFF)

BFF?

Comment (ytqsl) [reply embc-app/ClientApp/src/app/volunteer/pages/referral-view/referral-view.component.ts]: Backend For Frontend

Comment (severinbeauvais) [reply embc-app/ClientApp/src/app/volunteer/pages/referral-view/referral-view.component.ts]: > Backend For Frontend

I think you just made that up! :)

**Pull request merged**